Gopkg.toml

```diff
@@ -23,8 +23,9 @@ required = [
   version = "v1.15.5"
 
 [[override]]
-name = "sigs.k8s.io/cluster-api"
-revision = "0734939e05aeb64ab198e3feeee8b4e90ee5cbb2"
+  name = "sigs.k8s.io/cluster-api"
+  branch = "master"
+  #revision = "0734939e05aeb64ab198e3feeee8b4e90ee5cbb2"
```

Comment:
Is there a reason we don't pin a specific SHA?

Reply:
I have always though the point of dep was to specify only the minimal set of deps and have it to figure out the rest. Pinning the deps was the last resort. Cluster-api does not change so much so imho it's still fine to keep the latest master HEAD.

Reply:
Specific commit is determined by the dep, see [1]. So it's there, it's just picked for us.

[1] https://github.com/openshift/cluster-api-provider-aws/pull/127/commits/44661f6ee1eec5ecb7b3e079cce05c7097ae2b86#diff-bd247e83efc3c45ae9e8c47233249f18R961